release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -50,6 +50,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-13-installation-gcp-shielded-vms"]
+==== Installing a cluster on GCP using Shielded VMs
+In {product-title} {product-version}, you can use Shielded VMs when installing your cluster. Shielded VMs have extra security features including secure boot, firmware and integrity monitoring, and rootkit detection. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-configuration-parameters_installing-gcp-customizations[Installing a cluster on GCP with customizations] and Google's documentation on link:https://cloud.google.com/shielded-vm[Shielded VMs].
```

Comment:
How about replacing "_Shielded VMs have extra security features including secure boot, firmware and integrity monitoring, and rootkit detection_" with "_Shielded VMs leverage advanced platform security capabilities such as secure and measured boot, a virtual trusted platform module (vTPM), UEFI firmware, and integrity monitoring_" (which is from the mentioned Google doc)?

Reply:
I wanted to avoid directly copying Google's documentation into our documentation while still providing a decent summary of what shielded VMs are. Was my summary accurate?

Reply:
Got it, thanks!